### ISSUE TITLE
comments inlined with functions cause a syntax error

### ISSUE THREAD

**Issue by RaniSargees** [mathialo/bython#14]
```
def test(x): #prints x
     print(x)
```

is converted to 
```
def test(x) #prints x
{
    print(x);
}
```
which when run is converted to
```
def text(x) #prints x:
    print(x)
```
which causes a syntax error because the : is commented out.

**Comment by mathialo** [comment]
This is an interesting bug, thanks for noticing and reporting!

**Comment by b1f6c1c4** [comment]
@mathialo Could you please include `testcase/*` in the git repo? It's considered a good software engineering practice and would be VERY beneficial for debugging.

**Comment by mathialo** [comment]
@b1f6c1c4 I have now cleaned up and included the testcases directory, thanks for the tip!

The issue is now solved (somewhat ugly, I admit) by commit d1637ab107b1f72b8ec7b19c1799892a208f4d67
